openpype/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,225 @@
+from maya import cmds
+
+from openpype.hosts.maya.api import lib, plugin
+from openpype.hosts.maya.api.alembic import ALEMBIC_ARGS
+from openpype.lib import (
+    BoolDef,
+    TextDef,
+    NumberDef,
+    EnumDef,
+    UISeparatorDef,
+    UILabelDef,
+)
+
+
+def _get_animation_attr_defs(cls):
+    """Get Animation generic ddefinitions."""
+    defs = lib.collect_animation_defs()
+    defs.extend(
+        [
+            BoolDef("farm", label="Submit to Farm"),
+            NumberDef("priority", label="Farm job Priority", default=50),
+            BoolDef("refresh", label="Refresh viewport during export"),
+            BoolDef("includeParentHierarchy", label="Include Parent Hierarchy"),
+            BoolDef("writeNormals", label="Write Normals"),
+        ]
+    )
+
+    return defs
+
+
+def _get_animation_abc_attr_defs(cls):
+    """Get definitions relating to Alembic.
+
+    Most of the Alembic Arguments are booleans, those are stored in a
+    `abc_boolean_args` attribute, the other ones are their own attriubte.
+
+    An admin can define in settings the default arguments, which are then not
+    modifiable by the person publishing, unless they are added to the Alembic
+    Overrides setting, which is mapped to `abc_args_overrides`.
+
+    We use a combination of the two above to only show a muiltiselection dropdown
+    for booleans, and disabling the non-boolean arguments on the interface.
+
+    There's also a new separator so it's clearer what belongs to common Animation
+    publishes versus what is Almebic specific.
+    """
+    abc_defs = None
+    abc_defs = [UISeparatorDef(), UILabelDef("Alembic Options")]
```

Comment:
Separators in the UI need to have a unique key to ensure they all show up if there are multiple - best to provide it with an identifier of sorts.
```suggestion
    abc_defs = [UISeparatorDef("sep_alembic_options"), UILabelDef("Alembic Options")]
```

Also see: https://github.com/ynput/OpenPype/pull/5827